launch-checklist.md

```diff
@@ -1 +1,20 @@
+# Replicache Launch Checklist
 
+Before you launch with Replicache in your product, it's a good idea to double-check that you have correctly covered all the small details of integration. This list can help you determine if you might have missed a detail, or deferred and then forgotten about it.
+
+- **JS SDK**
+  - if you wish to version the schema of the mutations in `Push` and the `clientView` returned in `Pull`, version information is provided in `ReplicacheOptions.schemaVersion`
+  - re-auth is handled for `Push` and `Pull` via `getPushAuth` and `getPullAuth`
+- **All endpoints**
+  - the user is authenticated and the `clientID` verified to belong to that user
+- **Push endpoint**
+  - the `lastMutationID` for a client is updated transactionally along with the pushed mutations' effects
+  - all mutations with `id`s less than the client's current `lastMutationID` are skipped
+  - all mutations with `id`s greater than the client's current `lastMutationID+1` are ignored
+  - if a temporary error is encountered that might be resolved on retry, it halts processing mutations
```

Comment:
I would phrase this more as "think about what you want the error handling policy to be" and link to a doc, either the push endpoint reference doc which discusses this or a dedicated doc on error handling.